### ISSUE TITLE
Unable to use fibers with 32 bit node

### ISSUE THREAD

**Issue by rosen-vladimirov** [laverdet/node-fibers#200]
Hi, I'm using node 0.10.35, 32-bit version on Windows 8.1 and with the latest changes it looks like the fibers.node binary is not usable. I've rebuild it and every time I make npm install in fibers folder, it checks the binary and says its corrupted. Also any code that is using fibers is not working at all (you can check "quick-test.js" - it is also not working.

We've investigated the issue and we believe the problem is in coroutine.cc, line 41:
```c++
if (!mbi.State & MEM_COMMIT) {
```
maybe it should be:
```c++
if (!(mbi.State & MEM_COMMIT)) {
```

**Comment by laverdet** [comment]
Yikes! Yeah that'll do it. I committed a fix and published a 1.0.5 release w/ updated Windows binaries. That should fix it for you!

**Comment by rosen-vladimirov** [comment]
@laverdet , thanks for this immediate fix. I've tried it and it looks like everything is working fine. Thanks again for your responsiveness.
